### DISCUSSION TITLE
git add README.md

### DISCUSSION THREAD

**Discussion by palsachennaiah** [learndevopsonline/learndevopsonline#1598] [Q&A] [answered]
![Screenshot 2023-11-12 231446](https://github.com/learndevopsonline/learndevopsonline/assets/148023299/f7df9b28-f7cf-443a-ba32-c15440838f5a)

**Comment by sreedharm07** [comment]
Hi @palsachennaiah 
if you want to add readme file add from editor and give git push.

**Reply by sreedharm07** [reply]
git init.
git add .
git commit -m "Add existing project files to Git"
git push -u -f origin master.

these are steps to add from git

**Reply by palsachennaiah** [reply, marked as answer]
Ok problem solve

**Reply by FormlessCreator** [reply]
How go to where you write the git add? Thank you.
